shell/process.go

```diff
@@ -44,6 +45,23 @@ func (p *Process) Run() error {
 		cmd.Dir = p.Command.Dir
 	}
 
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt,
+		syscall.SIGHUP,
+		syscall.SIGTERM,
+		syscall.SIGINT,
+		syscall.SIGQUIT)
+
+	go func() {
+		sig := <-signals
+		if cmd.Process != nil {
+			//fmt.Printf("Passing signal %v to sub-process %d\n", sig, cmd.Process.Pid)
+			cmd.Process.Signal(sig)
+		}
+	}()
+
+	defer signal.Stop(signals)
```

Comment:
This looks like it will send the first signal only? Perhaps we should forward all signals until closed? Something like:

```go
go func() {
  for sig := range signals {
    if cmd.Process != nil {
      cmd.Process.Signal(sig)
    }
  }
}()
defer signal.Stop(signals)
defer close(signals)
```

This should continue forwarding each signal until the channel is closed.